Vates/VatesAPI/src/vtkPeakMarkerFactory.cpp

```diff
@@ -123,61 +184,35 @@ namespace VATES
    // Acquire a scoped read-only lock to the workspace (prevent segfault from algos modifying ws)
     Mantid::Kernel::ReadLock lock(*m_workspace);
 
-    vtkEllipsoidTransformer ellipsoidTransformer;
-
-    const int resolution = 8;
-    double progressFactor = 1.0/double(numPeaks);
+    const int resolution = 100;
+    double progressFactor = 1.0 / static_cast<double>(numPeaks);
 
-    vtkAppendPolyData* appendFilter = vtkAppendPolyData::New();
+    vtkAppendPolyData *appendFilter = vtkAppendPolyData::New();
```

Comment:
I'd like to change this line, but changing it to `vtkNew<vtkAppendPolyData> appendFilter;` generates a runtime error, `pointer being freed was not allocated`. Any idea what I am doing wrong?